packages/lesswrong/components/admin/CurationNoticesItem.tsx

```diff
@@ -0,0 +1,192 @@
+import { registerComponent, Components, getFragment } from '../../lib/vulcan-lib';
+import { useTracking } from "../../lib/analyticsEvents";
+import { isFriendlyUI } from '@/themes/forumTheme';
+import { commentBodyStyles } from '../../themes/stylePiping'
+import { useCurrentUser } from '../common/withUser';
+import { useCreate } from '@/lib/crud/withCreate';
+import { commentDefaultToAlignment } from '@/lib/collections/comments/helpers';
+import { User } from '@sentry/node';
+import { useUpdate } from '@/lib/crud/withUpdate';
+
+const styles = (theme: ThemeType) => ({
+  root: {
+    border: theme.palette.border.commentBorder,
+    borderRadius: isFriendlyUI ? theme.borderRadius.small : undefined,
+    cursor: "default",
+    marginBottom: 20,
+    background: theme.palette.background.pageActiveAreaBackground,
+    padding: 12,
+    position: "relative",
+    paddingBottom: 24,
+  },
+  editButton: {
+    ...theme.typography.body2,
+    color: theme.palette.primary.main,
+    padding: "0px 16px",
+    fontSize: "16px",
+    position: "absolute",
+    right: 10,
+    top: 10,
+    textTransform: "uppercase",
+    "&:hover": {
+      opacity: 0.5,
+      },
+},
+  publishButton: {
+      ...theme.typography.body2,
+      color: theme.palette.primary.main,
+      padding: "0px 16px",
+      fontSize: "16px",
+      position: "absolute",
+      right: 10,
+      bottom: 10,
+      textTransform: "uppercase",
+      "&:hover": {
+        opacity: 0.5,
+      },
+  },
+  meta: {
+    "& > div": {
+      marginRight: 5,
+    },
+    position: "relative",
+    display: "flex",
+    flexWrap: "wrap",
+    alignItems: "center",
+    justifyContent: "space-between",
+    rowGap: "6px",
+    marginBottom: 8,
+    color: theme.palette.text.dim,
+    paddingTop: "0.6em",
+    marginRight: isFriendlyUI ? 40 : 20,
+
+    "& a:hover, & a:active": {
+      textDecoration: "none",
+      color: isFriendlyUI ? undefined : `${theme.palette.linkHover.dim} !important`,
+    },
+  },
+  postTitle: {
+    ...theme.typography.body2,
+    ...theme.typography.commentStyle,
+    color: theme.palette.link.dim2,
+  },
+  username: {
+    ...theme.typography.body2,
+    ...theme.typography.commentStyle,
+    fontWeight: 600,
+    marginLeft: 25,
+  },
+  rightGroup: {
+    display: "flex",
+    alignItems: "center",
+    gap: 10,
+  },
+  commentBody: {
+    ...commentBodyStyles(theme)
+  }
+});
+
+const { ContentItemBody, Button, BasicFormStyles, WrappedSmartForm } = Components;
+
+export const CurationNoticesItem = ({curationNotice, classes}: {
+  curationNotice: CurationNoticesFragment,
+  classes: ClassesType<typeof styles>
+}) => {
+  const { captureEvent } = useTracking(); //it is virtuous to add analytics tracking to new components
+
+  const [ edit, setEdit ] = useState<boolean>(false)
+
+  const { create } = useCreate({
+    collectionName: "Comments",
+    fragmentName: 'CommentsList'
+  });
+
+  const {mutate: UpdateCurrentCurationNotice} = useUpdate({
```

Comment:
don't PascalCase function names - should camelCase

Reply:
kk, done